src/bioregistry/curation/curated_papers.csv

```diff
@@ -1 +1,8 @@
-pmid, relevant, relevancy_type, notes
\ No newline at end of file
+pmid, relevant, relevancy_type, notes
```

Comment:
I didn't notice this file was originally created but we should remove all the spaces after commas. Further, all text in the notes column should be in double quotes, otherwise the file will not be parseable as valid CSV.